### ISSUE TITLE
Option to not send a logo

### ISSUE THREAD

**Issue by hsychla** [sciyoshi/redmine-slack#82]
Hi, 

we tend to set the logo directly in the hook config in slack. 
After doing that I removed the logo URL from the plugin's config but then the notifications do not have any logo at all.

I guess that in this case the plugin overwrites the logo with "no logo".

Would it be possible to add an option to not send a logo?

**Comment by sciyoshi** [comment]
@hsychla you're correct that the plugin should probably not send any logo if there's none configured. A PR for this would be welcome!

**Comment by peelman** [comment]
I can't duplicate this; if I remove the logo URL and just leave a blank field there, the icon I have set in the web hook works.

I do think it would be better to say `.blank?` instead of `.empty?` when checking the icon field, but I'm hesitant to submit a pull request for something that seems to be working.

Was this fixed accidentally and just never linked/closed?
